db/seeds.rb

```diff
@@ -332,6 +336,27 @@
   University.create(name:, description:, address:, city:, country:, url:, ranking:, domestic_course_fee:, international_course_fee:, application_deadline:, student_count:, dorm:, email_domain:)
 end
 
+puts "Manually patching Coordinates"
+@university = University.find(8)
+@university.latitude = 3.117750,
+@university.longitude = 101.636600
+
+@university = University.find(10)
+@university.latitude =2.937890,
+@university.longitude = 101.801860
+
+@university = University.find(13)
+@university.latitude = 6.457510,
+@university.longitude = 100.505455
+
+@university = University.find(16)
+@university.latitude = 3.075530,
+@university.longitude = 101.549310
+
+@university = University.find(17)
+@university.latitude = 1.297990,
+@university.longitude = 103.775063
```

Comment:
For this section, you shouldn't put the latitude and longitude here. The information won't be saved in the database. Using .find(id) is also a bit problematic because whenever you reseed the file, the university id will increase unless you do a db:drop. Try putting the lat and long as keys inside the universities hash for those universities that need them. Then, you can put latitude = detail.latitude before creating the university. Do let me know if it doesn't work

Reply:
Just tested this. University.find has to work based on id number. have tested on rails s and display page of show. current implementation now works